packages/moonstone/Scrollable/ScrollableNative.js

```diff
@@ -364,7 +364,7 @@ class ScrollableBaseNative extends Component {
 			isHorizontalScrollButtonFocused = horizontalScrollbarRef.current && horizontalScrollbarRef.current.isOneOfScrollButtonsFocused(),
 			isVerticalScrollButtonFocused = verticalScrollbarRef.current && verticalScrollbarRef.current.isOneOfScrollButtonsFocused();
 
-		if (focusedItem && !isHorizontalScrollButtonFocused && !isVerticalScrollButtonFocused) {
+		if (!Spotlight.isPaused() && focusedItem && !isHorizontalScrollButtonFocused && !isVerticalScrollButtonFocused) {
```

Comment:
@ybsung What about to remove checking code from `Scrollable` instead of adding this to native one? It looks safe to blur for touch inputs event if Spotlight is paused.

Reply:
We added this condition in `Scrollable` because we found some issue.
https://github.com/enactjs/enact/pull/2314

Reply:
👍